neuralpp/test/quick_tests/symbolic/context_simplifier_test.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+import sympy
+import z3
+import operator
+from neuralpp.symbolic.context_simplifier import ContextSimplifier
+from neuralpp.symbolic.sympy_expression import SymPyVariable, sympy_Cond
+from neuralpp.symbolic.z3_expression import Z3Variable, Z3SolverExpression
+from neuralpp.symbolic.constants import if_then_else
+
+
+def test_context_simplifier1():
+    si = ContextSimplifier()
+    sympy_x = SymPyVariable(sympy.symbols('x'), int)
+    x = Z3Variable(z3.Int('x'))
+    y = Z3Variable(z3.Int('y'))
+    context = Z3SolverExpression()
+    context = context & (x == y)  # parenthesis is necessary
+    context = context & (y > 4)
+    result = si.simplify(sympy_x > 3, context)
+    assert result.value  # which means result is simplified to "True"
+
+
+def test_context_simplifier2():
```

Comment:
I am thinking we need more unit tests since these are very simple. For example, tests involving the simplification of a bound variable to a constant or to a function application.
Another one: what happens to expression "x" if context is "y = 3 and x = y"?